jwt-authorizer/src/layer.rs

```diff
@@ -414,3 +483,36 @@ where
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{IntoLayer, JwtAuthorizer};
+
+    #[tokio::test]
+    async fn jwt_auth_to_layer() {
+        let auth1: JwtAuthorizer = JwtAuthorizer::from_secret("aaa");
+        let layer = auth1.into_layer().await;
+        assert!(layer.is_ok());
+    }
+
+    #[tokio::test]
+    async fn vec_to_layer() {
+        let auth1: JwtAuthorizer = JwtAuthorizer::from_secret("aaa");
+        let auth2: JwtAuthorizer = JwtAuthorizer::from_secret("bbb");
+        let av = vec![auth1, auth2];
+        let layer = av.into_layer().await;
+        assert!(layer.is_ok());
```

Comment:
```suggestion
        let _ = av.into_layer().await.unwrap();
```

Apart from being shorter it also means the actual error is printed when the result isn't ok rather then just the assert that it's not ok, which is helpful for debugging

Reply:
that's right!